misc/local_tests.conf.save

```diff
@@ -0,0 +1,17 @@
+# Baseline test file for officially included tests.
```

Comment:
file shoudl not be here

Reply:
Been removed from git and added to the .gitignore, if the issue is that it "should not even be stored there or generated", then please let me know why it is being placed there.